Marlin/src/lcd/language/language_de.h

```diff
@@ -126,9 +129,9 @@
 #define MSG_VMAX                            _UxGT("V max ") // space by purpose
 #define MSG_VMIN                            _UxGT("V min")
 #define MSG_VTRAV_MIN                       _UxGT("V min Leerfahrt")
-#define MSG_ACCELERATION                    _UxGT("Beschleunigung")
+#define MSG_ACCELERATION                    MSG_ACC
 #define MSG_AMAX                            _UxGT("A max ") // space by purpose
-#define MSG_A_RETRACT                       _UxGT("A Retract")
+#define MSG_A_RETRACT                       _UxGT("A Rückzug")
```

Comment:
IMHO it should be "Einzug" instead of "Rückzug".

Reply:
Einzug and Rückzug is same

Reply:
"Einzug" was used for other translations of "RETRACT", that's why I was wondering why this one should be "Rückzug"

Reply:
Müsste man schauen